tests/gis_tests/geoapp/test_functions.py

```diff
@@ -257,6 +257,29 @@ def test_force_polygon_cw(self):
         st = State.objects.annotate(force_polygon_cw=functions.ForcePolygonCW('poly')).get(name='Foo')
         self.assertEqual(rhr_rings, st.force_polygon_cw.coords)
 
+    @skipUnlessDBFeature("has_FromWKB_function")
+    def test_fromwkb(self):
+        g = Point(56.811078, 60.608647)
+        g2 = City.objects.values_list(
+            functions.FromWKB(
+                models.Value(
+                    bytes(g.wkb),
+                    output_field=models.BinaryField(),
+                )
+            ),
```

Comment:
As of 1e38f1191de21b6e96736f58df57dfb851a28c1f, `output_field` doesn't need to be defined here.

```suggestion
            functions.FromWKB(Value(g.wkb.tobytes())),
```